.babelrc

```diff
@@ -1,6 +1,11 @@
 {
-    "compact": false,
     "presets": [
-        "es2015"
+      ["env", {
```

Comment:
I guess at this point we can get rid of Babel and just ship ES6 code.

Reply:
Fully agree with you. 
Nonetheless, because I do not master all parts of the code and because I do not know the code coverage given by the tests, I suggest to keep this part as a last-line-of-defense until I setup a complete coverage.

By the way, do you think it might be an opportunity to introduce typescript while getting rid of babel?

Reply:
I'm not a big fan of TypeScript and prefer vanilla JS. However, it's completely up to you, if you feel that it will be more convenient to use TypeScript then go for it.